config/json/db.json

```diff
@@ -4,10 +4,19 @@
       "url": "/not-found",
       "state": "notFound",
       "template": "not-found",
-      "endpoint": "post",
-      "params": {
-        "id": 1
-      }
+      "request": false
+    },
+    {
+      "url": "/patterns",
```

Comment:
@andybar2 is there a way to include this in the router by default, instead of sending it form the api?